src/dockerflow/logging.py

```diff
@@ -47,8 +48,13 @@ class JsonLogFormatter(logging.Formatter):
         'relativeCreated', 'stack_info', 'thread', 'threadName'
     ))
 
-    def __init__(self, format=None, datefmt=None, logger_name='Dockerflow'):
-        super(JsonLogFormatter, self).__init__(format, datefmt)
+    def __init__(self, fmt=None, datefmt=None, style='%', logger_name='Dockerflow'):
+        parent_init = logging.Formatter.__init__
+        argspec = inspect.getargspec(parent_init)
```

Comment:
Out of curiosity: in which case would the `logging.Formatter` class have a different signature? (maybe we could add a comment)

Reply:
Uups, scratch that. I hadn't read your comment on the issue. Definitely worth adding a comment IMO :)